v0/destinations/clevertap/transform.js

```diff
@@ -109,6 +109,25 @@ const mapIdentifyPayload = (message, profile) => {
   return payload;
 };
 
+const mapAliasPayload = message => {
+  const payload = {
+    d: [
+      {
+        type: "profile",
+        profileData: { identity: message.userId },
+        ts: get(message, "traits.ts") || get(message, "context.traits.ts"),
+        identity: message.previousId
+      }
+    ]
+  };
+
+  // If timestamp is not in unix format
+  if (payload.d[0].ts && !Number(payload.d[0].ts)) {
```

Comment:
Do we always get valid timestamp here ? We are not handling timestamp conversion based exceptions.

Reply:
We have deferred the responsibility for timestamp to the user, for passing via `traits`. Also passing illegal values like arrays or text-strings in ts causes no failures, as ts gets passed as null and rest operations happen as it is.